support/supportFunctions.js

```diff
@@ -0,0 +1,144 @@
+const axios = require('axios').default;
+const chai = require('chai');
+const should = chai.should()
+
+const userUrl = 'https://petstore.swagger.io/v2/user'
+
+
+class supportFuctions {
+
+    async createNewUser(dataTable) {
+        const data = dataTable.hashes();
+        await axios.post(userUrl, {
+                "id": data[0].id,
+                "username": data[0].username,
+                "firstName": data[0].firstName,
+                "lastName": data[0].lastName,
+                "email": data[0].email,
+                "password": data[0].password,
+                "phone": data[0].phone,
+                "userStatus": data[0].userStatus
+            })
+            .then(function(response) {
+                response.status.should.be.equal(200)
+            })
+            .catch(function(error) {
+                throw new Error(
+                    `Cannot post, error: ${error.message}}`,
```

Comment:
Change this Cannot post into Cannot create new user
you can keep error: ${error.message}